Flaky python/tests/test_jax.py::test_conversion

https://github.com/AMICI-dev/AMICI/actions/runs/12019272418/job/33505489070:

```
FAILED python/tests/test_jax.py::test_conversion - AssertionError: 
Not equal to tolerance rtol=1e-05, atol=1e-05
field sllh does not match
Mismatched elements: 1 / 2 (50%)
Max absolute difference among violations: 1.87327994e-05
Max relative difference among violations: 2.20057975e-05
 ACTUAL: array([0.085145, 0.851285])
 DESIRED: array([0.085145, 0.851267])
 ```

--- a/python/tests/test_jax.py
+++ b/python/tests/test_jax.py
@@ -19,6 +19,10 @@ pysb = pytest.importorskip("pysb")
 jax.config.update("jax_enable_x64", True)
 
 
+ATOL_SIM = 1e-12
+RTOL_SIM = 1e-12
+
+
 def test_conversion():
     pysb.SelfExporter.cleanup()  # reset pysb
     pysb.SelfExporter.do_export = True
@@ -113,6 +117,8 @@ def _test_model(model_module, ts, p, k):
     amici_solver = amici_model.getSolver()
     amici_solver.setSensitivityMethod(amici.SensitivityMethod.forward)
     amici_solver.setSensitivityOrder(amici.SensitivityOrder.first)
+    amici_solver.setAbsoluteTolerance(ATOL_SIM)
+    amici_solver.setRelativeTolerance(RTOL_SIM)
     rs_amici = amici.runAmiciSimulations(amici_model, amici_solver, [edata])
 
     check_fields_jax(
@@ -156,7 +162,7 @@ def check_fields_jax(
         jnp.array(my),  # my
         jnp.array(iys),  # iys
         diffrax.Kvaerno5(),  # solver
-        diffrax.PIDController(atol=1e-8, rtol=1e-8),  # controller
+        diffrax.PIDController(atol=ATOL_SIM, rtol=RTOL_SIM),  # controller
         diffrax.RecursiveCheckpointAdjoint(),  # adjoint
         2**8,  # max_steps
     )

